langchain4j-core/src/main/java/dev/langchain4j/model/LambdaStreamingResponseHandler.java

```diff
@@ -22,6 +24,11 @@
  *       onPartialResponse(System.out::println);
  * model.chat("Why is the sky blue?",
  *       onPartialResponseAndError(System.out::println, Throwable::printStackTrace));
+ *
+ * // Blocking (new methods)
```

Comment:
Maybe remove the `(new methods)` part of this line, as it won't age well

Reply:
Right. I removed it.